### PR TITLE
Add tests for the records errors

### PR DESCRIPTION
**Pull request by MancunianSam**
There's a test to make sure the records page redirects to the error page when there is a checksum error and a check to make sure the record check success message page redirects to the error message if there's an error.

### CODE REVIEW

File: src/test/resources/features/FileChecks.feature [hunk 1]
```diff
@@ -61,3 +61,11 @@ Feature: File Checks Page
...
     And the logged out user attempts to access the records page
     Then the logged out user should be on the login page
+
+  Scenario: The user will see an error when there is a checksum mismatch
+    Given A logged out user
+    And an existing consignment for transferring body MOCK1
+    And an existing transfer agreement
+    And the checksum check has failed
+    When the user is logged in on the records page
+    Then the user will see the error summary One or more files you uploaded have failed our checks
```
Comment (TomJKing) [inline src/test/resources/features/FileChecks.feature:71]: Is it worth putting the error message wording in double quotes just to make it clear what should be displayed?

File: src/test/resources/features/RecordsResults.feature [hunk 2]
```diff
@@ -23,3 +23,10 @@ Feature: Record results Page
     When the user is logged in on the records results page
     Then the user should see a general service error "Sorry, there is a problem with the service"
 
+  Scenario: The user will see an error when trying to access file check results when there is a checksum mismatch
```
Comment (TomJKing) [inline src/test/resources/features/RecordsResults.feature:26]: Is it worth putting the error message wording in double quotes just to make it clear what should be displayed?

**Review by TomJKing — COMMENTED** [review]
Looks good. Minor comment

File: src/test/resources/features/FileChecks.feature (hunk 1 above)
Comment (MancunianSam) [reply src/test/resources/features/FileChecks.feature]: Yep, makes sense. I've updated this one and the other one.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**